content/en/docs/tasks/security/authz-http/index.md

```diff
@@ -32,178 +30,243 @@ the product page, you can see the following sections:
 When you refresh the page, the app shows different versions of reviews in the product page.
 The app presents the reviews in a round robin style: red stars, black stars, or no stars.
 
-## Enabling Istio authorization
-
-Run the following command to enable Istio authorization for the `default` namespace:
-
-{{< text_import file="examples/TestAuthorizationForHTTPServices.txt" snippet="enabling_istio_authorization.sh" >}}
-
-Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`). Now you should see
-`"RBAC: access denied"`. This is because Istio authorization is "deny by default", which means that you need to
-explicitly define access control policy to grant access to any service.
-
 {{< tip >}}
-There may be some delays due to caching and other propagation overhead.
+If you don't see the expected output in the browser, retry in a few more seconds
+as there may be some delays due to caching and other propagation overhead.
 {{< /tip >}}
 
-## Enforcing Namespace-level access control
-
-Using Istio authorization, you can easily setup namespace-level access control by specifying all (or a collection of) services
-in a namespace are accessible by services from another namespace.
-
-In our Bookinfo sample, the `productpage`, `reviews`, `details`, `ratings` services are deployed in the `default` namespace.
-The Istio components like `istio-ingressgateway` service are deployed in the `istio-system` namespace. We can define a policy that
-any service in the `default` namespace that has the `app` label set to one of the values of
-`productpage`, `details`, `reviews`, or `ratings`
-is accessible by services in the same namespace (i.e., `default`) and services in the `istio-system` namespace.
-
-Run the following command to create a namespace-level access control policy:
-
-{{< text_import file="examples/TestAuthorizationForHTTPServices.txt" snippet="enforcing_namespace_level_access_control_apply.sh" >}}
+## Enforce mesh-level access control
 
-Once applied, the policy has the following effects:
+Using Istio, you can easily setup mesh-level access control for all workloads in your mesh.
 
-*   Creates a `ServiceRole` `service-viewer` which allows read access to any service in the `default` namespace that has
-the `app` label
-set to one of the values `productpage`, `details`, `reviews`, or `ratings`. Note that there is a
-constraint specifying that
-the services must have one of the listed `app` labels.
+Run the following command to create a mesh-level access control policy `deny-all` policy
+with no workload selector in the root namespace `istio-system`, the policy will select all
```

Comment:
Should we add a link for root namespace reference (e.g this https://preliminary.istio.io/docs/reference/config/istio.mesh.v1alpha1/) ? It may also worth to mention `istio-system` is the value used in the "before you begin" section

Reply:
+1. Also, root namespace is configurable. Readers need to make sure that root namespace is not set to any other value. Maybe we can have a step here to check the value of the root namespace.

Reply:
Updated to link to mesh config.

The task assumes istio is installed with the basic quick start as a demo purpose, we can mention the root namespace is configurable but I don't think we need to check it in this task.